ui/admin/tests/acceptance/targets/list-test.js

```diff
@@ -253,4 +266,102 @@ module('Acceptance | targets | list', function (hooks) {
     assert.strictEqual(currentRouteName(), 'scopes.scope.sessions.index');
     assert.dom(selectors.TABLE_SESSIONS_ID(instances.session.id)).isVisible();
   });
+
+  test('targets table is sorted by `created_time` descending by default', async function (assert) {
+    this.server.schema.targets.all().destroy();
+    await visit(urls.projectScope);
+    const createdTimeToNameMapping = {};
+    CREATED_TIME_VALUES_ARRAY.forEach((value, index) => {
+      createdTimeToNameMapping[value] = NAME_VALUES_ARRAY[index];
+    });
+    faker.helpers.shuffle(CREATED_TIME_VALUES_ARRAY).forEach((value) => {
+      this.server.create('target', {
+        name: createdTimeToNameMapping[value],
+        created_time: value,
+        scope: instances.scopes.project,
+      });
+    });
+
+    await click(commonSelectors.HREF(urls.targets));
+
+    assert
+      .dom(commonSelectors.TABLE_ROWS)
+      .isVisible({ count: CREATED_TIME_VALUES_ARRAY.length });
+    NAME_VALUES_ARRAY.forEach((expected, index) => {
+      // nth-child index starts at 1
+      assert.dom(commonSelectors.TABLE_ROW(index + 1)).containsText(expected);
+    });
+  });
+
+  test.each(
```

Comment:
Nice test parameterization! I'm always a big fan of this

Reply:
I didn't know about this actually, it was @hashicc who provided an example of this!